src/routes/highlights.route.js

```diff
@@ -0,0 +1,28 @@
+import { Router } from 'express';
+
+import {
+  createHighlight,
+  getHighlights,
+  deleteHighlights,
+} from '../routes/controllers/highlights.controller';
+import checkAuthorizedUser from './middlewares/authorized-user.middleware';
+
+const highlightRouter = Router();
+
+highlightRouter.post(
```

Comment:
There should be a documentation for this.

Reply:
Yes... It was an oversight. Would work on it ASAP.